pom.xml

```diff
@@ -58,6 +58,13 @@
     </snapshotRepository>
   </distributionManagement>
 
+  <repositories>
+    <repository>
+      <id>osgiify</id>
+      <url>file://c:/testrepo</url>
```

Comment:
Just FTR this looks to be optimized for Windows ;-)

Reply:
This needs to be replaced with the repo later, It's just my test-repo with the osgiified bundles. Commited accidentally :D